cmd/controller-manager/app/options/options.go

```diff
@@ -5,26 +5,17 @@ import (
 
 	"github.com/spf13/pflag"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/tools/leaderelection/resourcelock"
-	componentbaseconfig "k8s.io/component-base/config"
 	"k8s.io/klog/v2"
 )
 
-var (
-	defaultElectionLeaseDuration = metav1.Duration{Duration: 15 * time.Second}
-	defaultElectionRenewDeadline = metav1.Duration{Duration: 10 * time.Second}
-	defaultElectionRetryPeriod   = metav1.Duration{Duration: 2 * time.Second}
-)
-
 const (
 	defaultBindAddress = "0.0.0.0"
 	defaultPort        = 10357
 )
 
 // Options contains everything necessary to create and run controller-manager.
 type Options struct {
-	HostNamespace  string
-	LeaderElection componentbaseconfig.LeaderElectionConfiguration
```

Comment:
remove this unused `LeaderElection` config

Reply:
That's fine to me. We can add it back if there is a need in the future.